packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss

```diff
@@ -273,6 +273,10 @@
     }
 
     .Loader {
+        @include mobile {
+            visibility: hidden;
```

Comment:
This makes an element invisible, but it still takes place in the layout.
Is this intended?

Reply:
That was a part of an earlier implementation. Tested it with display: none and that indeed looks like a better alternative.